tsconfig.json

```diff
@@ -1,6 +1,8 @@
 {
   "compilerOptions": {
     /* Language and Environment */
+    // We only support node 18+, but v16 still works. We'll target v16 until it becomes a burden.
```

Comment:
Are these comments legal?

Reply:
@awaterma yes, `tsconfig.json` is [processed as a jsonc file](https://www.typescriptlang.org/docs/handbook/declaration-files/dts-from-js.html#tsconfig) (json w/ comments) so this is legal.